spinta/datasets/backends/wsdl/components.py

```diff
@@ -0,0 +1,31 @@
+from __future__ import annotations
+import contextlib
+import logging
+
+from requests import RequestException
+from zeep import Client as ZeepClient
+from zeep.exceptions import Error as ZeepError
+
+from spinta.datasets.components import ExternalBackend
+
+log = logging.getLogger(__name__)
+
+
+class WsdlBackend(ExternalBackend):
+    type: str = "wsdl"
+    client: ZeepClient | None
+
+    @contextlib.contextmanager
+    def begin(self):
+        self.client = self.get_client()
+        yield
+
+    def get_client(self) -> ZeepClient | None:
+        client = None
+        wsdl_url = self.config.get("dsn")
+        try:
+            client = ZeepClient(wsdl=wsdl_url)
+        except (RequestException, ZeepError) as err:
+            log.error(f"Zeep client initialization with url %s failed. Error: %s", wsdl_url,  err)
```

Comment:
Using the following approach you can quickly deduct which of the variables will be in which place. Otherwise, passing the variables as additional arguments, will have you looking left-to-right multiple times to check which of the varialbes `%s` maps to.

Well, at least that's what I think. Other than that, nor flake8, nor mypy is enabled, so can't think of any other reason not to use the following approach, let me know if I am just inexperienced:d

```suggestion
            log.error(f"Zeep client initialization with url {wsdl_url} failed. Error: {err}")
```